src/bin/enqueue-job.rs

```diff
@@ -1,17 +1,18 @@
-use cargo_registry::util::{CargoError, CargoResult};
+use cargo_registry::util::{human, CargoError, CargoResult};
 use cargo_registry::{db, tasks};
 use std::env::args;
 use swirl::Job;
 
 fn main() -> CargoResult<()> {
-    let conn = db::connect_now()?;
-
     match &*args().nth(1).unwrap_or_default() {
-        "update_downloads" => tasks::update_downloads()
-            .enqueue(&conn)
-            .map_err(|e| CargoError::from_std_error(e))?,
-        other => panic!("Unrecognized job type `{}`", other),
-    };
+        "update_downloads" => enqueue(tasks::update_downloads()),
```

Comment:
Could this be `enqueue(job, &conn)` instead? I want to get rid of the need for the `.map_err` eventually, which would mean we could replace the custom function with `use swirl::Job::enqueue;`

Reply:
Sure. The main reason I moved the connection inside the function is to save some typing (and maybe not to connect to the DB at all if the job name is invalid).